openstack/neutron/templates/etc/_uwsgi.ini.tpl

```diff
@@ -45,6 +45,7 @@ harakiri = 120
 harakiri-verbose = true
 post-buffering = 4096
 backlog-status = true
+py-tracebacker = /var/lib/neutron/uwsgi_pytracebacker.
```

Comment:
A little comment explaining that this is the file prefix could be helpful to future readers being confused, like I was, about the period at the end.

Reply:
I added an explanation to the commit message.